src/table/utils/accessibility-utils.ts

```diff
@@ -216,9 +216,12 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (value: string) => {
+export const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
+  const value = isHeadCell
```

Comment:
This looks like a potential bugg. As isHeadCell can be `undefined`, when `String(value)` is called. A `"undefined"` string will be copied.

I think should should just not attempt to copy the `value` if `value` is not a string.